From 3036b453793ea6f61006cef329d2280f5b66a69a Mon Sep 17 00:00:00 2001
From: Prabhu Sundararaj <prabhu.sundararaj@freescale.com>
Date: Thu, 12 Mar 2015 16:46:38 -0500
Subject: [PATCH 0896/1594] MGS-569 [#1650] MX6+ Update GPU AXI cache
 atttribute

commit 980a0ca6d8b7853a8dd8e8625b25c97ef51666cb from
git://git.freescale.com/imx/linux-2.6-imx.git

Add the new GPUs to the list to set AXI cache attribute.

Date: Mar 12, 2015

Signed-off-by: Prabhu Sundararaj <prabhu.sundararaj@freescale.com>

Signed-off-by: Xianzhong <b07117@freescale.com>
Acked-by: Jason Liu
---
 .../hal/kernel/arch/gc_hal_kernel_hardware.c       |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/mxc/gpu-viv/hal/kernel/arch/gc_hal_kernel_hardware.c b/drivers/mxc/gpu-viv/hal/kernel/arch/gc_hal_kernel_hardware.c
index 3a225a9..a0d5081 100644
--- a/drivers/mxc/gpu-viv/hal/kernel/arch/gc_hal_kernel_hardware.c
+++ b/drivers/mxc/gpu-viv/hal/kernel/arch/gc_hal_kernel_hardware.c
@@ -1601,7 +1601,9 @@ gckHARDWARE_InitializeHardware(
     }
 
     if (_IsHardwareMatch(Hardware, gcv2000, 0x5108)
+     || _IsHardwareMatch(Hardware, gcv2000, 0xffff5450)
      || _IsHardwareMatch(Hardware, gcv320, 0x5007)
+     || _IsHardwareMatch(Hardware, gcv320, 0x5303)
      || _IsHardwareMatch(Hardware, gcv880, 0x5106)
      || _IsHardwareMatch(Hardware, gcv400, 0x4645)
     )
-- 
1.7.5.4

